src/frontend/utils/dato.ts

```diff
@@ -15,20 +15,61 @@ export interface IPeriode {
 
 export const dagensDato = startOfToday();
 
-interface FormatterProps {
+export const tidenesMorgen = new Date(1000, 1, 1);
+
+export const tidenesEnde = new Date(3000, 1, 1);
+
+interface FormatterDateProps {
     dato?: Date;
     datoformat: Datoformat;
```

Comment:
Her er det en god blanding av engelsk og norsk 🙈. Tenker vi bør være konsekvente på ting vi lager selv, så enten burde interfacet hete noe à la `DatoFormatererProps` eller så bør feltene hete typ `dateformat`?

Reply:
Eller ønsker vi å beholde Date siden klassen vi jobber på heter det? 
Da burde vi kalle feltet `dato?: Date` -> `date?: Date`

Reply:
Har tenkt at det er fint å beholde Date i navnet så man er tydelig på hvilken type det er. Men usikker på om det er beste løsningen da 🤷‍♂️ kan isåfall endre fra dato til date

Reply:
Tenker det er nice siden Date er et konsept